core/domain/email_manager.py

```diff
@@ -544,6 +547,59 @@ def send_role_notification_email(
         sender_name=inviter_user_settings.username)
 
 
+def send_emails_to_subscribers(creator_id, exploration_id, exploration_title):
+    """Sends an email to all the subscribers of the creators when the creator
+    publishes an exploration.
+
+    Args:
+        creator_id: str. The id of the creator who has published an exploration
+            and to whose subscribers we are sending emails.
+        exploration_id: str. The id of the exploration which the creator has
+            published.
+        exploration_title: str. The title of the exploration which the creator
+            has published.
+    """
+
+    creator_name = user_services.get_username(creator_id)
+    email_subject = ('%s has published another exploration!' % creator_name)
+    email_body_template = (
+        'Hi %s,<br>'
+        '<br>'
+        'Your subscription, %s has published a new exploration. You can play '
```

Comment:
Drop "Your subscription, ". Also, maybe add the exploration title? E.g.

    %s has published a new exploration, Exploration Title. Play it here: ...